### PR TITLE
Add Tracy profile page

### PR DESCRIPTION
**Pull request by ftrader**
Add Tracy Chen biography page

### CODE REVIEW

**Review by merc1er — APPROVED** [review]
Working fine! Only thing that's a little bit strange is having two languages in the same page (could be done via i18n).

File: app/members/tracy-chen.html [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+    <section class="bg-light">
+      <div class="container py-5">
+        <div class="row my-5 justify-content-center">
+          <div class="col-md-9 text-lg">
+            <p>Tracy is the representative of BCHN, whose main role is
+            to establish contact channels and maintain communication on
+            behalf of BCHN team, BCH users and all ecological parties
+            (such as the exchange, mining pool, and wallet) to ensure
+            BCHN’s decisions to be fully approved and verified. She also
```
Comment (emergent-reasons) [inline app/members/tracy-chen.html:29]: Tracy is BCHN's public representative.
Her main role is to establish and maintain communication channels between the BCHN team and BCH users, stakeholders and the larger ecosystem such as exchanges, mining pools and wallets.
Tracy's work ensures that BCHN's decisions are communicated and vetted well with stakeholders.
She also...

File: app/members/tracy-chen.html [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+            runs BCHN's community and SNS in China to update the latest
+            project progress and financial reports.
+            She used to be a traditional media professional; and has
+            begun to join the encryption industry since 2018. She has
```
Comment (emergent-reasons) [inline app/members/tracy-chen.html:33]: She has worked as a traditional media professional and entered the cryptocurrency industry in 2018.
She has...

File: app/members/tracy-chen.html (hunk 1 above)
Comment (ftrader) [reply app/members/tracy-chen.html]: Thanks, updated

File: app/members/tracy-chen.html (hunk 2 above)
Comment (ftrader) [reply app/members/tracy-chen.html]: Thanks, updated

**Review by merc1er — APPROVED** [review]

**Pull request merged**